spec/google_visualr/base_chart_spec.rb

```diff
@@ -10,12 +10,50 @@
   describe "#initialize" do
     it "works" do
       @chart.data_table.should == @dt
+      @chart.version.should    == GoogleVisualr::BaseChart::DEFAULT_VERSION
+      @chart.material.should   == false
       @chart.options.should    == { "legend" => "Test Chart", "width" => 800, "is3D" => true }
     end
+
+    it "accepts material and version attributes" do
+      @chart = GoogleVisualr::BaseChart.new(@dt, { :version => "1.1", :material => true })
```

Comment:
Line is too long. [90/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.